db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150225023703) do
+ActiveRecord::Schema.define(version: 20150226012450) do
 
```

Comment:
not sure about that schema number, seems reallly random. but you must've really thought through every number in there. good for you! :1234: